aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -494,15 +499,21 @@ mod test {
                 &add_op,
                 AggregatorReadMode::Aggregated
             ),
-            Err(VMStatus::MoveAbort(_, EBOUND_OVERFLOW))
+            Err(VMStatus::ExecutionFailure {
+                status_code: StatusCode::DELAYED_FIELDS_SPECULATIVE_ABORT_ERROR,
+                ..
```

Comment:
So, we use the same status code for both underflows and overflows. Is there a way for VM to distinguish those? There is no sub_code set here.

Reply:
these are speculative errors - it means transaction execution is not valid - so not something that will ever be committed.

aggregators_v2.move errors are the ones user will see - and there we differentiate them